components/ias-adapter/go.mod

```diff
@@ -0,0 +1,114 @@
+module github.com/kyma-incubator/compass/components/ias-adapter
+
+go 1.20
```

Comment:
The go version is 1.20 here but 1.18 in the Makefile. I'm not sure if there will be a problem when some of the make targets are used? Did you try to execute `deploy-on-k3d` for example to see if everything is working as expected?